pulp_rpm/app/models.py

```diff
@@ -52,6 +52,16 @@
 log = getLogger(__name__)
 
 
+class JSONFieldWrapper(JSONField):
+    """A wrapper that implements a method for explicit data conversion."""
+
+    def from_db_value(self, value, expression, connection):
+        """Convert data retrieved from the database if they are type of the string."""
+        if isinstance(value, str):
+            return json.loads(value)
+        return value
```

Comment:
Nice! :clap:

Reply:
Thanks to @ipanova, I found out that it is not necessary to perform the conversion if we are storing the data as JSON binary. The same type is going to be retrieved from the database after all.